app/src/main/java/com/cornellappdev/android/eatery/Repository.java

```diff
@@ -22,6 +23,14 @@ public void setEateryList(ArrayList<EateryBaseModel> eateryList) {
     return mEateryList;
   }
 
+  public void setCtEateryList(ArrayList<EateryBaseModel> ctEateryList) {
+    this.mCtEateryList = ctEateryList;
+  }
+
+  public ArrayList<EateryBaseModel> getCtEateryList() {
+    return mCtEateryList;
```

Comment:
probably should be `mCollegetownEateryList`, it's more readable

Reply:
I thought mCollegetownEateryList was a bit lengthy and since everyone kinda knows what ct stands for..

Reply:
I vote mCollegetownEateryList

Reply:
Applied comment. Resolved.